test/09_contracts.ts

```diff
@@ -179,7 +179,10 @@ describe('contracts', () => {
 			expect(dbDeviceTypes).to.have.length(14);
 			expect(newDt).to.not.be.undefined;
 			expect(finDt).to.have.property('name', 'Fin');
-			expect(finDt).to.have.property('contract', JSON.stringify(finDtContract));
+			expect(finDt).to.have.deep.property(
+				'contract',
+				JSON.parse(JSON.stringify(finDtContract)),
```

Comment:
This is necessary to verify the serialization and deserialization correctly because comparing two JSON objects in Chai doesn't ignore fields that are set to "undefined", but the serialization handles this.